guides/source/association_basics.md

```diff
@@ -559,6 +565,10 @@ class CreateEmployees < ActiveRecord::Migration[7.1]
 end
 ```
 
+NOTE: The `to_table` option passed to `foreign_key` and more are explained in [`SchemaStatements#add_reference`][connection.add_reference].
```

Comment:
`more` refers to the options?
```suggestion
NOTE: The `to_table` option passed to `foreign_key` and more options are explained in [`SchemaStatements#add_reference`][connection.add_reference].
```

Reply:
`options` is good, I was just thinking more info in general though.

There are lots of opportunities to tie the guides and API closer together, I think where there is duplication between the two we should be better about navigating the reader to the API reference for more detailed information instead.